### PR TITLE
docs(breadcrumb): Removed description from the docsite

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
We had discussion with the design team. Description should be moved to the Fluent2 website.

## Previous Behavior
All the description was on the docsite
image

## New Behavior
Description and best practices are removed.
image

### CODE REVIEW

File: packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbDescription.md [hunk 1]
```diff
@@ -1,3 +1 @@
-Breadcrumbs should be used as a navigational aid in your app or site. They indicate the current page's location within a hierarchy and help the user understand where they are in relation to the rest of that hierarchy.
-
-Navigational breadcrumbs can also provide one-click access to higher levels of that hierarchy.
+For detailed guidance on when and how to use this component, go to Fluent 2 Breadcrumbs.
```
Comment (ValentinaKozlova) [inline packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbDescription.md:1]: The link will be added when Fluent2 website has Breadcrumb

**Review by tudorpopams — APPROVED** [review]
LGTM! 🚢

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/82476dd4c1c1aaa2a482e531ad0f8b31da3d3d8f'>82476dd4c1c1aaa2a482e531ad0f8b31da3d3d8f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Pull request merged**